src/ml_board/frontend/Dashboard/src/components/graphs/Graph.tsx

```diff
@@ -0,0 +1,76 @@
+import { Box, Card, Grid } from "@mui/material";
+import { ChartOptions } from "chart.js";
+import { Line } from "react-chartjs-2";
+import { useAppSelector } from "../../app/hooks";
+import { selectChartLabelsById, selectExperimentsPerChartById } from "../../redux/charts/chartsSlice";
+import { selectColorMap } from "../../redux/status/statusSlice";
+// styles
+import styles from "./Graphs.module.css";
+
+// https://www.chartjs.org/docs/latest/general/data-structures.html
+
+export default function Graph({ chart_id }: { chart_id: string }) {
+
+    const chartLabels = useAppSelector(state => selectChartLabelsById(state, chart_id));
+    const experimentsDic = useAppSelector(state => selectExperimentsPerChartById(state, chart_id));
```

Comment:
experimentsDict ?

Reply:
it is a dictionary of experiments? or do you want me to add the `t` to the name?

Reply:
yeah please add the t. Sorry for being so nitpicking

Reply:
Done!